app/authentication/auth.py

```diff
@@ -137,10 +137,19 @@ def requires_auth():
 
 
 def _auth_by_api_key(auth_token):
+    # TODO: uncomment this when the grace period for the token prefix is over
```

Comment:
This will be the eventual validation we enable. I wrote it now while everything was fresh and then commented it out. If anyone prefers this to be omitted, just let me know.